### PR TITLE
docs: add tidb_enable_amend_pessimistic_txn depreciation

### PR DESCRIPTION
**Pull request by benmaoer**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1033,6 +1033,9 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 > - 启用该参数仅意味着一阶段提交成为可选的事务提交模式，实际由 TiDB 自行判断选择最合适的提交模式进行事务提交。
 
 ### `tidb_enable_amend_pessimistic_txn` <span class="version-mark">从 v4.0.7 版本开始引入</span>
+> **警告：**
+>
+> 从 v6.6 版本开始，该变量将被废弃，默认使用 [`元数据锁`](/metadata-lock.md) 机制解决 `Information schema is changed` 报错。
```
Comment (qiancai) [inline system-variables.md:1038]:
```suggestion

> **警告：**
>
> 从 v6.6 版本开始，该变量被废弃，TiDB 将默认使用 [`元数据锁`](/metadata-lock.md) 机制解决 `Information schema is changed` 报错。
```

**Review by wjhuang2016 — APPROVED** [review]

File: system-variables.md [hunk 2]
```diff
@@ -1034,6 +1034,10 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 ### `tidb_enable_amend_pessimistic_txn` <span class="version-mark">从 v4.0.7 版本开始引入</span>
 
+> **警告：**
+>
+> 从 v6.6 开始，该变量将被废弃，TiDB 会默认使用 [`元数据锁`](/metadata-lock.md) 机制解决 `Information schema is changed` 报错。
```
Comment (Oreoxmt) [inline system-variables.md:1039]:
```suggestion
> 从 v6.6 开始，该变量将被废弃，TiDB 会默认使用 [元数据锁](/metadata-lock.md) 机制解决 `Information schema is changed` 报错问题。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: system-variables.md (hunk 2 above)
Comment (qiancai) [inline system-variables.md:1039]:
```suggestion
> 从 v6.6.0 开始，该变量将被废弃，TiDB 会默认使用 [`元数据锁`](/metadata-lock.md) 机制解决 `Information schema is changed` 报错。
```

**Review by qiancai — APPROVED** [review]

File: system-variables.md [hunk 3]
```diff
@@ -1034,6 +1034,10 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 ### `tidb_enable_amend_pessimistic_txn` <span class="version-mark">从 v4.0.7 版本开始引入</span>
 
+> **警告：**
+>
+> 从 v6.6.0 开始，该变量将被废弃，TiDB 会默认使用 [`元数据锁`](/metadata-lock.md) 机制解决 `Information schema is changed` 报错。
```
Comment (Oreoxmt) [inline system-variables.md:1039]:
```suggestion
> 从 v6.6.0 开始，该变量将被废弃，TiDB 会默认使用 [元数据锁](/metadata-lock.md) 机制解决 `Information schema is changed` 报错的问题。
```

File: system-variables.md [hunk 4]
```diff
@@ -1034,6 +1034,10 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 ### `tidb_enable_amend_pessimistic_txn` <span class="version-mark">从 v4.0.7 版本开始引入</span>
 
+> **警告：**
+>
+> 从 v6.6.0 开始，该变量将被废弃，TiDB 会默认使用 [元数据锁](/metadata-lock.md) 机制解决 `Information schema is changed` 报错的问题。
```
Comment (qiancai) [inline system-variables.md:1039]:
```suggestion
> 从 v6.6.0 开始，该变量将被废弃，TiDB 会默认使用[元数据锁](/metadata-lock.md) 机制解决 `Information schema is changed` 报错的问题。
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**